pages/common/mv.md

```diff
@@ -17,19 +17,19 @@
 
 - Do not prompt for confirmation before overwriting existing files:
 
-`mv -f {{path/to/source}} {{path/to/target}}`
+`mv --force {{path/to/source}} {{path/to/target}}`
 
 - Prompt for confirmation before overwriting existing files, regardless of file permissions:
 
-`mv -i {{path/to/source}} {{path/to/target}}`
+`mv --interactive {{path/to/source}} {{path/to/target}}`
 
 - Do not overwrite existing files at the target:
```

Comment:
This is the first time I'm seeing this notation. Why not make it `[n]ot`?

Reply:
> This is the first time I'm seeing this notation. Why not make it `[n]ot`?

In this case, I think enclosing "not" with the notation would be contextually incorrect since the long option is for "no clobber".

We often use this format in translations (optional) to port or show the short option mnemonics since it is not ideal to highlight them within the native words.